Tasks/app-store-release/task.json

```diff
@@ -75,15 +76,15 @@
             "label": "Bundle ID",
             "defaultValue": "",
             "required": true,
-            "helpMarkDown": "The unique app identifier (e.g. com.myapp.etc)."
+            "helpMarkDown": "The unique app identifier (e.g. com.myapp.etc). Required if the track is Production."
         },
         {
             "name": "ipaPath",
             "type": "filePath",
             "label": "Binary Path",
             "defaultValue": "",
-            "required": true,
-            "helpMarkDown": "Path to the binary to publish (i.e. IPA)."
+            "required": false,
```

Comment:
Because I (accidentally) changed the wrong 'required' property.  (It should be "Bundle ID".)  Great catch!